attributes/user.rb

```diff
@@ -4,4 +4,6 @@
   user['group'] = 'ajones'
   user['gid'] = '1000'
   user['home'] = '/home/ajones'
+  user['home']['manage?'] = true
```

Comment:
(additionally you may wish to consider using `chef_rewind` to add the manage attribute yourself without any upstream changes)